pyre/extra_tests/parity_tests/exc_match_fold_subclass_of_one_kind.py

```
@@ -0,0 +1,73 @@
+    # class the frame naming the clauses never mentions.
+    raise exc
+
+
+def step(exc):
+    try:
+        read_header(exc)
+    except EndOfArchive:
+        return "end"
+    except HeaderError:
+        return "header"
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)